### ISSUE TITLE
win32 usage?

### ISSUE THREAD

**Issue by hsdk123** [luochong/Cocos2dxVideo#1]
Hi, this seems like a great project - I was wondering if this could be used with the win32 version of cocos2d-x.

**Comment by WangYanqing** [comment]
Of course, I'm using it playing *.flv videos on cocos2d win32 simulator.

**Comment by Bilalmirza** [comment]
Hi!
Can I use this to play video on macOs and win32?
